ibr/png-parser/CreateIBR.py

```diff
@@ -0,0 +1,34 @@
+import ibr_pb2
+import sys
+from boundary_extraction.boundary_extraction import extract_boundary
+from imagedata_creation.imagedata_creation import get_image_data
+from utils import *
```

Comment:
let's avoid `import *`, either `import utils` and then reference methods like `utils.set_visualization_image_data` or explicitly import each method 'from utils import set_visualization_image_data'